### DISCUSSION TITLE
Stellungnahme von Hoymiles zu HM-Serie

### DISCUSSION THREAD

**Discussion by Manos1966** [hoylabs/OpenDTU-OnBattery#928] [Show and tell]
Interessant... dieser Satz: "Ab **August 2023** stellten wir Produktion und Vertrieb der **gesamten HM-Serie** in EU sofort ein"

Es ging um das Model HM-800, aber die Stellungnahme von HOYMILES spricht von der gesamten HM-Serie.

> Ein Wechselrichter der Marke Hoymiles fiel in einem Test der Stiftung Warentest aufgrund mangelnder elektromagnetischer Verträglichkeit auf. Die Bundesnetzagentur hat das Gerät vorübergehend vom Markt genommen.
Ein aktueller [Balkonkraftwerk-Test](https://www.test.de/balkonkraftwerk-test-5501703-0/) der Stiftung Warentest kam zu einem ernüchternden Ergebnis: Nur eine von acht getesteten Mini-PV-Anlagen schnitt mit der Note "Gut" ab. Im Test fiel vor allem der Wechselrichter HM-800 der Marke Hoymiles negativ auf. Das Problem: Laut Stiftung Warentest hat das Gerät eine mangelhafte elektromagnetische Verträglichkeit (EMV). Dadurch könnte der Wechselrichter elektromagnetische Störungen, zum Beispiel Probleme mit der Internetverbindung oder dem Radioempfang, verursachen. **Aus diesem Grund hat die Bundesnetzagentur den Hoymiles HM-800 vorläufig vom Markt genommen, so die Stiftung Warentest.**

![Stellungnahme-von-Hoymiles_v20240426-1320x1866](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/129660360/94ec8cc7-2a15-445d-84c6-05d3bb95b2bf)

**Comment by Harisjems** [comment]
Alles, was aus China und Russland kommt, ist für Spionage und Weltuntergang verantwortlich. :-)
Ich würde meine HM-Serie erst tauschen, wenn sie kaputt geht.

**Comment by gitisgreat2023** [comment]
Mmm.... ist die E-Star Energy HERF-1800 auch betroffen?

**Comment by Manos1966** [comment]
Bezüglich des EMV Tests, ein paar Details:
Balkonkraftwerke - Hat Warentest hier LACK gesoffen?
https://www.youtube.com/watch?v=CZ6ML4xwMEA
